notebooks/216-attention-center/216-attention-center.ipynb

```diff
@@ -61,8 +61,7 @@
     "from pathlib import Path\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        ov.save_model(model, ir_model_path.as_posix())
</pre><p>no need in as_posix</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1270/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>removed</p>